### PR TITLE
835239  UG document for the spline range area series in .net ma UI

### PR DESCRIPTION
**Pull request by EzhilarasanElangovan31**
**Description** 

Added Marker and its customisation content for spline range area series, 
Added spline range area series images

### CODE REVIEW

**Comment by VimalaThirumalaiKumarSF3739** [comment]
Fine

File: MAUI/Cartesian-Charts/SplineRangeArea.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ this.Content = chart;
 
 {% endtabs %}
 
+![Spline range area chart type in MAUI Chart](Chart-types-images/maui_spline_range_area_chart.png)
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/SplineRangeArea.md:71]: The type image not have difference from default. change the image.

Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/SplineRangeArea.md:71]: Changed the type image

**Review by Saravanan-Madhesh — APPROVED** [review]
Approved, but spline type image not good.

**Pull request merged**